### ISSUE TITLE
Media Sources and phpThumbOf

### ISSUE THREAD

**Issue by hephaestoskin** [splittingred/phpThumbOf#52]
It seems that use of phpThumbOf with meda sources is a bit of a headache, I need a custom output filter to append te URL before processing in order for it to function. That is in a getResources Tpl.

**Comment by atmmarketing** [comment]
I agree with this, I cannot seem to get phpThumbOf to work without an absolute path from the hard drive root to the image.
